Wallet.cpp error on windows build

This is my mistake. Since this #3828 was my first pull request doesn't look like workflows ever ran to catch this. Will patch today and update my forks to run ci.

<details>
<summary>
logs
</summary>
<code>
 D:/a/monero-gui/monero-gui/src/libwalletqt/Wallet.cpp:859:5: error: 'u_int64_t' was not declared in this scope; did you mean 'uint64_t'?
</code>
<br />
<code>
  859 |     u_int64_t total;
</code>
<br />
<code>
      |     ^~~~~~~~~
</code>
<br />
<code>
      |     uint64_t
</code>
<br/>
<code>
D:/a/monero-gui/monero-gui/src/libwalletqt/Wallet.cpp:860:14: error: expected ';' before 'spent'
</code>
<br/>
<code>
  860 |     u_int64_t spent;
</code>
</details>

--- a/src/libwalletqt/Wallet.cpp
+++ b/src/libwalletqt/Wallet.cpp
@@ -856,8 +856,8 @@ Q_INVOKABLE QString Wallet::getReserveProof(bool all, quint32 account_index, qui
 Q_INVOKABLE QString Wallet::checkReserveProof(const QString &address, const QString &message, const QString &signature) const
 {
     bool good;
-    u_int64_t total;
-    u_int64_t spent;
+    uint64_t total;
+    uint64_t spent;
     bool success = m_walletImpl->checkReserveProof(address.toStdString(), message.toStdString(), signature.toStdString(), good, total, spent);
     std::string result = std::string(success ? "true" : "false") + "|" + std::string(good ? "true" : "false") + "|" + QString::number(total).toStdString() + "|" + QString::number(spent).toStdString();
     return QString::fromStdString(result);

